tests/test_authentication.py

```diff
@@ -207,3 +207,62 @@ def test_login_required_roles_debuggable_but_still_requires_roles(
             mock_request.location
             == "https://authenticator/service/user?roles_required=COMMENTER"
         )
+
+    def test_login_required_with_return_app_redirects_to_signed_out_without_token(
+        self, flask_test_client
+    ):
+        """
+        GIVEN a flask_test_client and
+            route decorated with @login_required decorator with the "return_app" parameter set to "post-award-frontend"
+        WHEN a request is made without any "fsd-user-token" cookie
```

Comment:
This file can be named to something more specific e.g. test_return_app.py?

Reply:
This file tests the whole of authentication, I've just added 3 tests at the bottom specific to the `return_app` changes